### DISCUSSION TITLE
Blurred wallpaper as background

### DISCUSSION THREAD

**Discussion by SunsetTechuila** [nimsandu/spicetify-bloom#164] [Ideas]
Is it possible to use [this package](https://github.com/sindresorhus/wallpaper)? If so, we can simulate Mica or Acrilic on backdrop instead of solid color

**Comment by nimsandu** [comment]
@SunsetTechuila It would be great
Ill try

**Reply by SunsetTechuila** [reply]
Cider uses it for [Mica](https://github.com/ciderapp/Cider/blob/c15f55d0ee9383edceebc937349ba99cb55cd6a8/src/renderer/main/mica.js)

**Reply by nimsandu** [reply]
@SunsetTechuila Well thanks 
Could you please include every resource that's useful on implementing this?

Because there are experts who are willing to help us.
So it will be easier to make this come trough sooner

**Reply by SunsetTechuila** [reply]
Also I know that [vibrancy continued](https://github.com/illixion/vscode-vibrancy-continued) uses [electron-acrylic-window](https://github.com/Seo-Rii/electron-acrylic-window) for transparency
As for Mica in Cider - I left a link to the code for it. I really don't know anything about web developing, so I can't explain more than you will understand from the code

**Comment by SunsetTechuila** [comment]
Impossible because it [uses](https://github.com/sindresorhus/wallpaper/blob/9c065213cbf549c740c7fd0c9b24e099a7b099bc/index.js#L1) NodeJS API, which Spicetify [doesn't support](https://spicetify.app/docs/development/js-modules#:~:text=In%20other%20words%2C%20that%20Javascript%20should%20not%20use%20require%20function%20to%20include%20other%20packages%20or%20NodeJS%20API)

**Comment by SunsetTechuila** [comment]
And electron-acrylic-window is totally irrelevant to this case

**Reply by SunsetTechuila** [reply]
I wasn't very smart a week ago
